src/servers/ZoneServer2016/entities/trapentity.ts

```diff
@@ -243,4 +245,31 @@ export class TrapEntity extends BaseSimpleNpc {
         return isInsideCube(Array.from(position) as Point3D, this.cubebounds);
     }
   }
+
+  OnProjectileHit(server: ZoneServer2016, damageInfo: DamageInfo) {
+    const damage = damageInfo.damage * 6; // bullets do more to damage traps
```

Comment:
should do less imo

Reply:
You reckon so? What would be a good damage multiplier?

Reply:
idk actually lets try with this and see players reactions